### ISSUE TITLE
Composer setup not possible

### ISSUE THREAD

**Issue by chaos234** [dwoo-project/dwoo#42]
Hey,

I can't install dwoo over composer on my win10 machine with the supplied composer command because there is no option or any hint - also not on googles side - how to change from stable to dev so it's not possible to install it over composer.

Please fix this according to the latest composer cmd line parameters.

Oh ... composer require --dev dwoo/dwoo does also not work.

**Comment by typoworx-de** [comment]
Any update on this? Issue still persists.

```
[InvalidArgumentException]                                                   
  Could not find package dwoo/dwoo at any version for your minimum-stability   
  (stable). Check the package spelling or your minimum-stability
```

Cannot install/require. The Package is still listed at packagist:
https://packagist.org/packages/dwoo/dwoo

**Comment by emulienfou** [comment]
Problem fixed on 1.2.1.
Packagist/Composer now refer to project hosted on GitLab: https://gitlab.com/dwooproject/dwoo/

You can try `composer require dwoo/dwoo`.
However GitLab authentication is required, try to fix this problem!
